apiary.apib

```diff
@@ -2049,6 +2051,11 @@ Get details for a single invoice.
                         + rate: 0.21 (number)
                         + taxable (Money)
                         + tax (Money)
+                + withholding_taxes (array)
```

Comment:
Will be weird for the countries where it does not apply to have this in the response.